docker-compose.yml

```diff
@@ -93,9 +95,10 @@ services:
     volumes:
       - ./src:/usr/src/app/src
       - ./style:/usr/src/app/style
+      - ./.env:/usr/src/app/.env
```

Comment:
Was this working before without bind-mounting `.env`?

Reply:
Yes, this should be working on the main branch. The old Dockerfile had [this line](https://github.com/conda-incubator/conda-store-ui/blob/main/Dockerfile#L15)
```
    if [[ ! -f ".env" ]]; then mv .env.example .env; fi;
```

which bakes whatever `.env` file (or the `.env.example` file if `.env` is not present) on your local system into the image when you run a `docker build`.

This line had to be removed in this pr because:
* if you are using docker compose to develop you'll need to rebuild the image in order to see your changes. By mounting the file in you just need to restart the webserver/docker image
* we don't want a `.env` file baked into a prod image

Reply:
Oh that's a great improvement to be able to update the `.env` during runtime and to keep it out of the production image (for security)